protostar/git.py

```diff
@@ -0,0 +1,215 @@
+import subprocess
+import re
+import os.path
+from typing import Dict, Optional
+from pathlib import Path
+from functools import wraps
+
+from protostar.utils.package_info import PackageInfo
+from protostar.protostar_exception import ProtostarException
+
+# pylint: disable=subprocess-run-check
+
+# set to true when debugging
+GIT_VERBOSE = False
+SHARED_KWARGS = (
+    {"check": True}
+    if GIT_VERBOSE
+    else {
+        "check": True,
+        "stdout": subprocess.DEVNULL,
+        "stderr": subprocess.DEVNULL,
+    }
+)
+
+CREDENTIALS = [
+    "-c",
+    'user.name="Protostar"',
+    "-c",
+    'user.email="protostar@protostar.protostar"',
+]
```

Comment:
I think we should use the user credentials.
(Next.js uses user credentials for the first commit.)

Reply:
I'm not sure whether this will work because we pipe stdout to dev/null